components/VM_Arm/src/main.c

```diff
@@ -1156,6 +1157,9 @@ int main_continued(void)
     ZF_LOGF_IF(err, "Failed to bind CB to SID");
 #endif /* CONFIG_ARM_SMMU */
 
+    vm.fdt_ori = (void *)ps_io_fdt_get(&_io_ops.io_fdt);
```

Comment:
Will we place all places where `ps_io_fdt_get()` is calles by `vm->fdt_ori` then also? E.g. in `load_linux()` above?